pkg/kubelet/util/util_windows.go

```diff
@@ -112,3 +113,15 @@ func LocalEndpoint(path, file string) string {
 	}
 	return u.String() + "//./pipe/" + file
 }
+
+var tickCount = syscall.NewLazyDLL("kernel32.dll").NewProc("GetTickCount64")
+
+// GetBootTime returns the time at which the machine was started, truncated to the nearest second
+func GetBootTime() (time.Time, error) {
```

Comment:
/cc @kubernetes/sig-windows-misc

Reply:
cc @PatrickLang

Reply:
@alinbalutoiu  can you also please take a look. the code looks good at a high level

Reply:
@michmike looks good to me, it is working fine as well.

Reply:
excellent. thank you!